README.md

```diff
@@ -225,7 +225,7 @@ Cookies.get('name'); // => 'value'
 Cookies.remove('name', { secure: true });
 ```
 
-## Converters
+## Converters <a name="converter"></a>
```

Comment:
As explained in the [submitted review](https://github.com/js-cookie/js-cookie/pull/291#pullrequestreview-17196201), if you remove `<a name="converter"></a>`, then the link will just work.